package.json

```diff
@@ -6,7 +6,14 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jake testAll",
+    "testAll": "jake testAll",
+    "testUnit": "jake testUnit",
+    "testIntegration": "jake testIntegration",
```

Comment:
Can get away with just using npm commands without this package or is there a limitation that this solves?

Reply:
I am doing this because I want to build the next app before running the integration tests. I don't know if there is a cross platform way to run two commands in sequence from just the npm commands.

Reply:
I've switched over to using `run-s` from `npm-run-all`